packages/roadworks-cancelled-notification/index.ts

```diff
@@ -0,0 +1,172 @@
+                                               ></path></svg
+                                             ><span style="padding-left: 5px;"
+                                               >GOV.UK</span
+                                             ></span
+                                           ></a
+                                         >
+                                       </div>
+                                     </div>
+                                   </header>
+                                  <div style="font-family: Arial, sans-serif; padding-left: 30px">
+                       <h1>Notification of cancelled Street Manager roadworks</h1>
+                       <p>There has been a cancelled roadwork update from Street manager. This may effect the end date showing in your disruption. Please review the roadwork and edit the disruption accordingly.The following disruptions are attached to cancelled roadworks;</p>
+                       ${disruptionLinks.map((link, i) => `<a href=${link}>Link to disruption ${i + 1}</a>`).join(" ")}
```

Comment:
Maybe modify this join to be `.join(`<br/>`)` so that links are displayed on new lines instead

Reply:
done